src/coreclr/vm/dllimport.h

```diff
@@ -56,6 +56,15 @@ struct StubSigDesc
         }
     }
 #endif // _DEBUG
+    
+#ifndef DACCESS_COMPILE
+    void InitTypeContext(TypeHandle* classInst, DWORD classInstCount, TypeHandle* methodInst, DWORD methodInstCount)
+    {
+        LIMITED_METHOD_CONTRACT;
+
+        m_typeContext = SigTypeContext(Instantiation(classInst, classInstCount), Instantiation(methodInst, methodInstCount));
```

Comment:
We should assert the current state of `m_typeContext` if possible. I assume it should be `IsEmpty()` is `TRUE`?